UPGRADE-4.2.md

```diff
@@ -10,6 +10,7 @@ Config
 ------
 
  * Deprecated constructing a `TreeBuilder` without passing root node information.
+ * Deprecated `FileLoaderLoadException`. Renamed to `LoaderLoadException`.
```

Comment:
we need an entry in the `UPGRADE-5.0.md` file too mentioning the removal of the `FileLoaderLoadException`

Reply:
done